### DISCUSSION TITLE
Example of using graphite queries with Grafonnet

### DISCUSSION THREAD

**Discussion by unjuli** [grafana/grafonnet#100] [Q&A] [answered]
Could someone kindly share a simple example code for using Graphite queries with Grafonnet? Thanks.

**Comment by unjuli** [comment]
@malcolmholmes @Duologic Could you kindly assist with this? I'm unable to transform the the prometheus query examples shared to a Graphite query. Some example of a simple query and a graphite datasource like `Graphite Prod` would be extremely helpful.

**Comment by unjuli** [comment]
Do I first need to add a new veneer type for graphite [here](https://github.com/grafana/grafonnet/blob/main/grafonnet-base/veneer/query.libsonnet#L20) before it can be used as a query type ?

**Comment by Duologic** [comment, marked as answer]
Grafonnet doesn't currently support Graphite queries. We don't have a timeline for that yet.
